src/mpl/include/mpl_proc_mutex_posix.h

```diff
@@ -0,0 +1,125 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *
+ *  (C) 2001 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+/*
+ * Interprocess Mutex
+ */
+#ifndef MPL_PROC_MUTEX_POSIX_H_INCLUDED
+#define MPL_PROC_MUTEX_POSIX_H_INCLUDED
+
+#include <errno.h>
+#include <pthread.h>
+
+typedef pthread_mutex_t MPL_proc_mutex_t;
+
+#if defined(MPL_NEEDS_PTHREAD_MUTEXATTR_SETTYPE_DECL)
+int pthread_mutexattr_settype(pthread_mutexattr_t * attr, int kind);
+#endif /* MPL_NEEDS_PTHREAD_MUTEXATTR_SETTYPE_DECL */
+
+/* FIXME: convert errors to an MPL_PROC_MUTEX_ERR value */
```

Comment:
Can you go ahead and do this instead of adding a fixme? This seems like a small thing to me, but if it's a big effort, let me know.

Reply:
I do not see any difficulty to add it. I copied from the pthread header file. I will update it.